keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -270,22 +245,26 @@ The control plane:
   - Provide a way to preempt the current leader.
   - Reuse the existing lease mechanism as much as possible.
 
-### Component Identity Leases
+### Component Lease Candidates
 
-Components will create identity leases similar to those used by apiserver
-identity, e.g.:
+Components will create lease candidates similar to those used by apiserver
+identity. Some key differences are certain fields like `LeaseTransitions` and `HolderIdentity` are removed.
+See the API section for the full API.
```

Comment:
we have to specify how we get from a `NoCoordination` lease to the candidate's strategy. Does the controller match candidates even in `NoCoordination` mode of the lease and set the strategy in case a new component is elected? Or is it the duty of the client-go code?

Reply:
I'll clarify a bit more in the KEP but essentially the controller will read NoCoordination and do nothing. If the lease is already held by a nocoordination component, it will be renewed indefinitely so there's no point doing extra work. When the leader component releases the lease, the controller will then reconcile the `Lease`, see that the lease is currently not held, fetch the `LeaseCandidate`s and update the strategy.